### PR TITLE
[Docs] Update deprecation dates

### PR DESCRIPTION
**Pull request by pedrosousa**
Update deprecation dates:
* Firewall rules, rate limiting rules (legacy), WAF managed rules (legacy): 2025-01-15.
* Brotli toggle: 2024-08-15.
* Mobile Redirect: 2024-06-30.
* Page Rules: remove section about API deprecation.

### CODE REVIEW

File: content/fundamentals/api/reference/deprecations.md [hunk 1]
```diff
@@ -8,20 +8,78 @@ weight: 5
 
 Cloudflare occasionally makes updates to our APIs that result in behavior changes or deprecations. When this happens, we will communicate when the API will no longer be available and whether there will be a replacement.
 
-## Page Rules
-**End of life date: Jan 6th, 2025**
+## Firewall Rules API and Filters API
```
Comment (pedrosousa) [inline content/fundamentals/api/reference/deprecations.md:11]: Moved section to the top of the page due to the updated sunset date.

File: content/fundamentals/api/reference/deprecations.md [hunk 2]
```diff
@@ -8,20 +8,78 @@ weight: 5
...
+- POST /zones/:zone_id/filters
+- PUT /zones/:zone_id/filters
+- DELETE /zones/:zone_id/filters
+- GET /zones/:zone_id/filters/:filter_id
+- PUT /zones/:zone_id/filters/:filter_id
+- DELETE /zones/:zone_id/filters/:filter_id
+
+Replacement: [WAF custom rules](/waf/custom-rules/)
+
+## WAF managed rules APIs (previous version)
```
Comment (pedrosousa) [inline content/fundamentals/api/reference/deprecations.md:36]: Moved section to the top of the page due to the updated sunset date.

File: content/fundamentals/api/reference/deprecations.md [hunk 3]
```diff
@@ -8,20 +8,78 @@ weight: 5
...
+- PATCH /zones/:zone_id/firewall/waf/packages/:package_id/rules/:rule_id
+- GET /zones/:zone_id/firewall/waf/overrides
+- POST /zones/:zone_id/firewall/waf/overrides
+- GET /zones/:zone_id/firewall/waf/overrides/:override_id
+- PUT /zones/:zone_id/firewall/waf/overrides/:override_id
+- DELETE /zones/:zone_id/firewall/waf/overrides/:override_id
+
+Replacement: [WAF Managed Rules](/waf/managed-rules/) (new version)
+
+## Rate Limiting API (previous version)
```
Comment (pedrosousa) [inline content/fundamentals/api/reference/deprecations.md:59]: Moved section to the top of the page due to the updated sunset date.

File: content/fundamentals/api/reference/deprecations.md [hunk 4]
```diff
@@ -8,20 +8,78 @@ weight: 5
 
 Cloudflare occasionally makes updates to our APIs that result in behavior changes or deprecations. When this happens, we will communicate when the API will no longer be available and whether there will be a replacement.
 
-## Page Rules
```
Comment (pedrosousa) [inline content/fundamentals/api/reference/deprecations.md:11]: This section was removed for now.

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**